docs/getting_started/install_manually.md

```diff
@@ -1,29 +1,284 @@
+``` bash
+git clone https://github.com/ezsystems/ezplatform.git /var/www/ezplatform
+```
+
+You can check out a tag, or use the `master` branch if you are interested in working with the latest version.
+
+!!! tip
+
+    You can use any other folder name in place of `ezplatform` in the examples above.
+    You'll point your virtual host to this folder to use as its root.
+
+## 4. Create a database
+
+Create a new database (you can substitute `ezplatform` with the database name you want to use, but keep it in mind in next steps):
+
+``` bash
+mysql -u root -e 'CREATE DATABASE ezplatform CHARACTER SET utf8;'
+```
+
+## 5. Run installation scripts
+
+Composer will look inside the `composer.json` file and install all of the packages required to run eZ Platform.
+The `app/console` script will then install eZ Platform for your desired environment (dev/prod).
+
+### a. Run composer install
+
+From the folder into which you downloaded eZ Platform, run:
+
+``` bash
+php -d memory_limit=-1 composer install
+```
+
+Once the installer gets to the point that it creates `app/config/parameters.yml`, you will be presented with a few decisions:
+
+1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); choose any random string you like, made up of characters, numbers, and symbols, up to around 32 characters. This is used by Symfony when generating [CSRF tokens](http://symfony.com/doc/current/book/forms.html#forms-csrf), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](http://symfony.com/doc/current/book/http_cache.html#edge-side-includes).
+2. You can accept the default options for `database_driver`, `database_host` and `database_port`
+3. For `database_name` and `database_user`, reoplace them if you customized those values during configuration.
+4. If you set a password for your database user, enter it when prompted for `database_password`.
+
+The installer should continue once you've completed this manual portion of the installation process.
+
+### b. Run eZ Platform's installer
+
+``` bash
+php app/console ezplatform:install --env=prod clean
+```
+
+In this example the `ezplatform:install` script uses the `clean` installation type in production environment.
+
+If Composer asks for your token, you must log in to your GitHub account generate a new token
+(edit your profile, go to Developer settings > Personal access tokens and Generate new token with default settings).
+Be aware that the token will be shown only once, so do not refresh the page until you paste the token into the Composer prompt.
+This operation is performed only once when you install eZ Platform for the first time.
+
+!!! enterprise
+
+    ##### Enable Date-based Publisher
+
+    To enable delayed publishing of Content using the Date-based publisher, you need to set up cron to run the command `app/console ezstudio:scheduled:publish` periodically.
```

Comment:
date-based?

Reply:
Changed to capitalizing whole of "Date-based Publisher", becuase it's a feature name, like Flex Workflow for example.